app/v3/gov/proposal_community_tax_usage.go

```diff
@@ -121,7 +123,13 @@ func (tp *CommunityTaxUsageProposal) Validate(ctx sdk.Context, k Keeper, verify
 		return err
 	}
 
-	if tp.TaxUsage.Usage != UsageTypeBurn {
+	taxCoins := k.ck.GetCoins(ctx, auth.CommunityTaxCoinsAccAddr)
+	if !taxCoins.IsAllGTE(tp.TaxUsage.Amount) {
+		return ErrNotEnoughCommunityTax(k.codespace, taxCoins, tp.TaxUsage.Amount)
+	}
```

Comment:
duplicate check，`AllocateFeeTax`  has checked

Reply:
It is also executed on submitting proposal.